core/domain/user_services.py

```diff
@@ -132,6 +135,7 @@ def __init__(
                 preference.
             preferred_audio_language_code: str or None. Default language used
                 for audio translations preference.
+            display_alias: str or None. Name of a user display in Android UI.
```

Comment:
``Display name of a user logged into the Android app.``

Reply:
Done.